projects/core/components/primitive-calendar/test/primitive-calendar.component.spec.ts

```diff
@@ -382,11 +439,20 @@ describe('integration with TUI_FIRST_DAY_OF_WEEK token', () => {
         return daysOfWeekContainers.map(container => container.nativeElement.textContent);
     }
 
-    function getColumnDates(columnIndex: number): string[] {
-        const columnDatesContainers =
+    function getColumnCells(columnIndex: number): DebugElement[] {
+        return (
             fixture.debugElement.queryAll(
                 By.css(`.row:not(.row_weekday) .cell:nth-child(${columnIndex + 1})`),
-            ) || [];
+            ) || []
+        );
+    }
+
+    function isAllWeekdays($cells: DebugElement[]): boolean {
+        return $cells.every(cell => cell.classes['cell_weekend']);
```

Comment:
why are you using `$` notation ?) 
just interested to know

Reply:
just to show that it is an element: that it has classes, styles and etc

almost like jquery notation